ts/src/pro/coinex.ts

```diff
@@ -21,22 +19,24 @@ export default class coinex extends coinexRest {
                 'watchTicker': true,
                 'watchTickers': true,
                 'watchTrades': true,
-                'watchMyTrades': false, // can query but can't subscribe
+                'watchMyTrades': true,
                 'watchOrders': true,
                 'watchOrderBook': true,
-                'watchOHLCV': true, // only for swap markets
-                'fetchOHLCVWs': true,
+                'watchOHLCV': false,
+                'fetchOHLCVWs': false,
```

Comment:
I think we can implement `watchBidsAsks` as well
![image](https://github.com/user-attachments/assets/537b81e8-c7e4-46b1-a62e-b429f22c260e)